src/actions/checkDockerInstallation.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from "../utils/logger";
+import { execCommand } from "../utils/execCommand";
+
+export async function checkDockerInstallation(): Promise<void> {
+  try {
+    const isInstalled = await execCommand("command -v docker");
+    if (!isInstalled) {
+      Logger.info("Docker is not installed. Installing Docker...");
+
+      await execCommand(`curl -fsSL https://get.docker.com -o get-docker.sh`);
+      await execCommand(`sh get-docker.sh`);
```

Comment:
also put the command in a variable to the top

Reply:
done in https://github.com/jaracogmbh/docker-compose-health-check-action/pull/2/commits/a358106562b9ec6fa7cb73c30e87f9aec4809ec1